flink-core/src/main/java/org/apache/flink/api/common/io/GenericCsvInputFormat.java

```diff
@@ -300,14 +300,14 @@ protected void setFieldsGeneric(boolean[] includedMask, Class<?>[] fieldTypes) {
 	// --------------------------------------------------------------------------------------------
 	//  Runtime methods
 	// --------------------------------------------------------------------------------------------
-	
+
 	@Override
-	public void open(FileInputSplit split) throws IOException {
-		super.open(split);
+	public void initializeSplit(FileInputSplit split, Long offset) throws IOException {
```

Comment:
nit: this can be protected (in other formats as well)

Reply:
I will fix this, merge it and open PR-s for backporting

Reply:
Do you think we should merge this to release-1.12 too?

Reply:
yes, I think we should fix this everywhere. it might be included in 1.12 if we have another RC